html/modules/custom/reliefweb_fields/components/reliefweb-user-posting-rights/reliefweb-user-posting-rights.js

```diff
@@ -84,6 +84,43 @@
       return label;
     },
 
+    /**
+     * Create a filter for users.
+     */
+    createUserSelect: function () {
```

Comment:
Not sure a user name `select` element is the best here. That doesn't allow to filter on the email address or user ID. It can also become difficult to use when there are many entries (50+) which was the complaint the editorial team has.

That said, they can use the built-in browser text search so maybe that's enough.

Reply:
The idea was to have a textbox and not a select?

Reply:
New filter is good, thanks.